content/docs/tasks/traffic-management/egress-gateway.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Configure an Egress Gateway
+description: Describes how to configure Istio to direct traffic to external services through a dedicated gateway service
+weight: 43
```

Comment:
Please add a keywords: line in here, which we now use in the site for cross-referencing and automaticaly creating See also sections. Check out other .md files in the repo for examples.

Reply:
@geeknoid done.